test/navbar-components/navigationBarTest.js

```diff
@@ -379,10 +379,8 @@ describe('navigationBar tests', function () {
           this.app.client
             .activateURLMode()
             .waitForExist(urlBarIcon)
-
-            // TODO: Fix
             .getAttribute(urlBarIcon, 'class').then((classes) =>
-              classes.includes('fa-unlock') && classes.includes('insecure-color')
+              classes.includes('fa-unlock')
```

Comment:
> Follow ups just add issues and PR's to the already long QA testing list

note: the manual tests are covered by `Test Plan (URL bar un/lock icon)` on the 1st post.